### PR TITLE
fix(classification): add default type in argparse

### LINKED ISSUE #90: add type in argparse
## 环境
1.系统环境：ubuntu
2.MegEngine版本：1.2.0
3.python版本：3.6
4.模型名称：/

## 复现步骤
1. python3 train.py --arch resnet50 --batch-size 32 --learning-rate 0.025 --ngpus 8 --save .

## 请提供关键的代码片段便于追查问题
https://github.com/MegEngine/Models/blob/master/official/vision/classification/resnet/train.py#L58

The default type of argparse is `str`, so once `batch_size`, `learning_rate` or `world_size` is modified, it will cause an error

### PR DESCRIPTION
**Pull request by zhouyizhuang-megvii**
resolve #90

also fix a bug that Server would be unexpectedly recycled.

### CODE REVIEW

File: official/vision/classification/shufflenet/test.py [hunk 1]
```diff
@@ -53,15 +53,15 @@ def main():
...
+    parser.add_argument("--dist-port", default=23456, type=int)
+    parser.add_argument("--world-size", default=1, type=int)
+    parser.add_argument("--rank", default=0, type=int)
 
     args = parser.parse_args()
 
     # create server if is master
     if args.rank <= 0:
-        dist.Server(port=args.dist_port)
+        server = dist.Server(port=args.dist_port)  # pylint: disable=unused-variable
```
Comment (FateScript) [inline official/vision/classification/shufflenet/test.py:64]: server是一个没有用过的变量，flake8跪了

File: official/vision/classification/shufflenet/train.py [hunk 2]
```diff
@@ -80,15 +87,15 @@ def main():
...
+    parser.add_argument("--dist-port", default=23456, type=int)
+    parser.add_argument("--world-size", default=1, type=int)
+    parser.add_argument("--rank", default=0, type=int)
 
     args = parser.parse_args()
 
     # create server if is master
     if args.rank <= 0:
-        dist.Server(port=args.dist_port)
+        server = dist.Server(port=args.dist_port)  # pylint: disable=unused-variable
```
Comment (FateScript) [inline official/vision/classification/shufflenet/train.py:98]: ditto

**Review by FateScript — CHANGES_REQUESTED** [review]

File: official/vision/classification/shufflenet/test.py (hunk 1 above)
Comment (zhouyizhuang-megvii) [reply official/vision/classification/shufflenet/test.py:64]: 我disable了，我看到的怎么是

`official/nlp/bert/mrpc_dataset.py:12:0: E0401: Unable to import 'tokenization' (import-error)`

**Review by FateScript — APPROVED** [review]

**Comment by FateScript** [comment]
LGTM.
